src/sidebar/components/new-note-btn.js

```diff
@@ -21,9 +31,9 @@ function NewNoteButton({ annotationsService, settings }) {
     }
     const annot = {
       target: [],
-      uri: topLevelFrame.uri,
+      uri: topLevelFrame ? topLevelFrame.uri : '',
```

Comment:
I wonder if this even a valid state when `mainFrame` returns null? but using a blank string seems reasonable.

Reply:
This component shouldn't be shown when `mainFrame` returns `null`. We definitely can't create an annotation in that situation, so I would suggest to just bail out of the callback in that case, or just make the component render nothing.